### PR TITLE
Move logging functions to log package

### PR DESCRIPTION
**Pull request by adombeck**
... and print all log messages to stderr.
    
We only keep using stdout for dialog output (questions etc. which require user input).

### CODE REVIEW

File: internal/cmd/create/create.go [hunk 1]
```diff
@@ -59,36 +60,36 @@ func run(cmd *cobra.Command, args []string, opts *cmdOpts) (err error) {
...
+			log.Errorf(err, "Unable to created fuzz test, file already exists %s\n", stubPath)
 			return cmdutils.WrapSilentError(err)
 		}
 	}
 
 	// show success message
-	dialog.Successf("Fuzz test stub created at %s\n", stubPath)
-	dialog.Info(`
+	log.Successf("Fuzz test stub created at %s\n", stubPath)
+	log.Info(`
```
Comment (jochil) [inline internal/cmd/create/create.go:92]: I don't see why something like this should go to `stdErr`... it is not diagnostic/debug output. Same for the `log.Success`

File: internal/cmd/create/create.go [hunk 2]
```diff
@@ -59,36 +60,36 @@ func run(cmd *cobra.Command, args []string, opts *cmdOpts) (err error) {
 	if err != nil {
 		return err
 	}
-	dialog.Debugf("Selected fuzz test type: %s\n", opts.testType)
+	log.Debugf("Selected fuzz test type: %s\n", opts.testType)
```
Comment (jochil) [inline internal/cmd/create/create.go:63]: `\n` can be removed due to the previous commit

Comment (adombeck) [reply internal/cmd/create/create.go]: Done

**Comment by adombeck** [comment]
@jochil I think you meant to approve this, no?

**Review by jochil — APPROVED** [review]

**Pull request merged**